fixed-hash/src/hash.rs

```diff
@@ -512,7 +512,7 @@ macro_rules! impl_rand_for_fixed_hash {
 
 			/// Assign `self` to a cryptographically random value.
 			pub fn randomize(&mut self) {
-				let mut rng = $crate::rand::rngs::EntropyRng::new();
+				let mut rng = $crate::rand::rngs::OsRng;
```

Comment:
does it work for wasm? see #130 for the previous switch

Reply:
thanks, probably not then

let me check

Reply:
Yes, it works

`$cargo build --target wasm32-unknown-unknown` builds fine for `fixed-hash`